presto-docs/src/main/sphinx/release/release-0.132.rst

```diff
@@ -7,3 +7,7 @@ Hive Changes
 
 * Change unit of Hive namenode latency JMX stats from seconds to milliseconds.
 * Fix ``NullPointerException`` when inserting a null value for a partition column.
+* Fix a correctness issue that can occur when any ``JOIN`` depends on the output
+  of another ``[LEFT | RIGHT | FULL OUTER] JOIN`` that has an inner side (or
```

Comment:
Actually, remove the formatting for "join" in both places. Then this can say
"... when any join depends ..."
".. of another outer join that has ..."